test/functionalTest/cases/0947_v2_list_all_entities/GET_v2_entities.test

```diff
@@ -156,83 +154,77 @@ Date: REGEX(.*)
 03. GET /v2/entities
 ====================
 HTTP/1.1 200 OK
-Content-Length: 288
+Content-Length: 422
 Content-Type: application/json
 Date: REGEX(.*)
 
 [
     {
         "A0": {
-            "type": "number", 
+            "metadata": {},
+            "type": "number",
             "value": "255"
-        }, 
-        "A1": "E1/T1/A1", 
+        },
+        "A1": {
+            "metadata": {},
+            "type": null,
+            "value": "E1/T1/A1"
+        },
         "A2": {
-            "type": "AT2", 
+            "metadata": {},
+            "type": "AT2",
             "value": "E1/T1/A2"
-        }, 
+        },
         "A3": {
-            "m1": "M1", 
-            "type": "a", 
+            "metadata": {
+                "m1": {
+                    "type": null,
+                    "value": "M1"
+                }
+            },
+            "type": "a",
             "value": "E1/T1/A3/M1"
-        }, 
+        },
         "A4": {
-            "m1": "M1", 
-            "m2": {
-                "type": "tm2", 
-                "value": "M2"
-            }, 
-            "m3": {
-                "type": "number", 
-                "value": "3"
-            }, 
-            "type": "a4", 
+            "metadata": {
+                "m1": {
+                    "type": null,
+                    "value": "M1"
+                },
+                "m2": {
+                    "type": "tm2",
+                    "value": "M2"
+                },
+                "m3": {
+                    "type": "number",
+                    "value": "3"
+                }
+            },
+            "type": "a4",
             "value": "E1/T1/A4/M1-3"
-        }, 
-        "id": "E1", 
+        },
+        "id": "E1",
         "type": "T1"
     }
 ]
 
 
-04. GET /v2/entities?options=count,normalized
-=============================================
+04. GET /v2/entities?options=count,keyValues
+============================================
```

Comment:
OK, this is also a 'big change' in a functest.
As the previous GET (03) was without options (normalized), I decided to exercise 'keyValues' in this one.

NTC